### PR TITLE
Export email address data from outlook 2013 for windows

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5198d51bf6/how-to/)

**Comment by stephamon** [comment]
My edits:

[Export.email.address.data.from.Outlook2013.for.Windows_SF.docx](https://github.com/rackerlabs/rackspace-how-to/files/1352698/Export.email.address.data.from.Outlook2013.for.Windows_SF.docx)
[Export.email.address.data.from.Outlook2013.for.Windows_SF.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1352697/Export.email.address.data.from.Outlook2013.for.Windows_SF.pdf)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5f461e52f0/how-to/)

File: content/rackspace-email/export-email-address-data-from-outlook-2013-for-windows.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    <img src="{% asset_path rackspace-email/export-email-address-data-from-outlook-2013-for-windows/export_to_file2013.png %}" />
+
+5. Click **Outlook Data File (.pst)**, and then click **Next**.
+
+    <img src="{% asset_path rackspace-email/export-email-address-data-from-outlook-2013-for-windows/outlook_data_file2013.png %}" />
+
+6. Select the folder that you want to export, and then click **Next**. 
+
+   Select the email address at the top of the folder list and then select the checkbox for **Include subfolders** to export all data.
```
Comment (catlook) [inline content/rackspace-email/export-email-address-data-from-outlook-2013-for-windows.md]: ... list, and then ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e5a0ade4c1/how-to/).

**Pull request merged**